README.md

```diff
@@ -36,14 +36,23 @@ ws.on('close', function () {
 
 ## API
 
-#### `const drive = new Hyperdrive(corestore, [key])`
+#### `const drive = new Hyperdrive(corestore, [key], [options])`
 
 Creates a new Hyperdrive instance.
 
 `corestore` must be an instance of `Corestore`.
 
 `key` should be a Hypercore public key. If you do not set this, Hyperdrive will use the core at `{ name: 'db' }` in the passed Corestore instance.
 
+Available `options`:
+```js
+{
+  name: 'db'
+}
+```
+
+You can use Corestore namespaces to store multiple Hyperdrives, or using a different name as well.
```

Comment:
Unsure about this explaination, while it's basically correct maybe it's not the best practice so we don't want it in the README?

Reply:
I meant about using a different name